src/System.Management.Automation/engine/Modules/ImportModuleCommand.cs

```diff
@@ -989,7 +989,27 @@ private PSModuleInfo ImportModule_LocallyViaName(ImportModuleOptions importModul
                 try
                 {
                     this.ArgumentList = new object[] { psSession };
-                    ImportModule_LocallyViaName(importModuleOptions, wildcardEscapedPsd1Path);
+
+                    // The correct module version has already been imported from the remote session and created locally. 
+                    // The locally created module always has a version of 1.0 regardless of the actual module version
+                    // imported from the remote session, and version checking is no longer needed and will not work while 
+                    // importing this created local module.
+                    Version originalBaseMinimumVersion = BaseMinimumVersion;
+                    Version originalBaseMaximumVersion = BaseMaximumVersion;
+                    Version originalBaseRequiredVersion = BaseRequiredVersion;
+                    BaseMinimumVersion = null;
+                    BaseMaximumVersion = null;
+                    BaseRequiredVersion = null;
+                    try
```

Comment:
You can just use the existing try/catch (for this.ArgumentList).  I don't think a new one is needed.

Reply:
Yes, didn't notice that.  Will fix.